packages/e2e/features/ui/components/authenticator/reset-password-angular.feature

```diff
@@ -0,0 +1,27 @@
+Feature: Reset Password
+
+  This will be used specifically for Angular users who have changed the translations for 'Send Code' for GH #1784.
```

Comment:
This is only for Angular, and will be removed in the next major version update, when we remove this legacy `hasTranslations` check.